src/components/message/styledMessages.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components/macro';
+
+export const Inner = styled.div`
+  display: flex;
+  align-items: flex-end;
+  flex-direction: ${({ direction }) => direction};
+  margin: auto;
+  width: 100%;
+  @media (max-width: 1000px) {
+    flex-direction: column;
+  }
+`;
+
+export const Bubble = styled.div`
+  padding: 30px 30px;
+  background-color: ${({ backgroundColor }) => backgroundColor};
```

Comment:
Seems like this would be way easier to do with vanilaa CSS and `:nth-child(even)`. I like your implimentation and it appears to work, but here's another method if you're interested: [this](https://stackoverflow.com/questions/45871439/before-and-after-pseudo-classes-used-with-styled-components)

Reply:
yup; I realized that too late but I can fix it for the code review